server/src/main/java/org/apache/iotdb/db/newsync/receiver/ReceiverService.java

```diff
@@ -218,7 +220,6 @@ public void start() throws StartupException {
 
   @Override
   public void stop() {
-    stopPipeServer();
     try {
       receiverManager.close();
```

Comment:
why delete stop pipe server

Reply:
`stop()` is called when deregister service such as exiting IoTDB. 

`stopPipeServer()` stops pipeserver and marks pipeserver unenabled to disk. Receiver service will read disk file and recover status, the correct status of pipeserver should not be unenabled.